server/log_rpc_server_test.go

```diff
@@ -539,12 +539,12 @@ func TestGetLeavesByLeafValueHashInvalidHash(t *testing.T) {
 
 	// Should have succeeded at RPC level
 	if err != nil {
-		t.Fatalf("GetLeavesByLeafValueHash() = %v", err)
+		t.Fatalf("GetLeavesByLeafValueHash(hash empty) = %v, want nil", err)
```

Comment:
This is still confusing (here and below) because the error messages don't distinguish between the two return values.